INSTALLATION.md

```diff
@@ -9,9 +9,9 @@ check the [TL;DR section](#tldr).
 
 ## Installing Atomkraft
 
-Installing Atomkraft requires [Python](https://www.python.org) 3.10, `git` and [`java` 17 or newer](https://apalache.informal.systems/docs/apalache/installation/jvm.html).
+Installing Atomkraft requires [Python](https://www.python.org) 3.8 or newer (we suggest using Python 3.10), together with its package manager [pip](https://pip.pypa.io/en/stable/installation/).
```

Comment:
We suggest the _latest_ version. Python's latest version is `3.11` now.

Reply:
Why Java requirement is removed? We need the latest Java runtime for Apalache.

Reply:
I'd be afraid of suggesting that (yet), given that we don't run any tests with `3.11`, nor we use it actively. (Or am I wrong about that?)